xml/templates/civicrm_navigation.tpl

```diff
@@ -352,7 +352,7 @@ VALUES
     ( @domainID, 'civicrm/admin/setting/component?reset=1',             '{ts escape="sql" skip="true"}Components{/ts}', 'Enable Components',                                    'administer CiviCRM', '', @systemSettingslastID, '1', NULL, 1 ),
     ( @domainID, 'civicrm/admin/extensions?reset=1',                    '{ts escape="sql" skip="true"}Extensions{/ts}', 'Manage Extensions',                                    'administer CiviCRM', '', @systemSettingslastID, '1', 1,    3 ),
 
-    ( @domainID, 'civicrm/admin/setting/updateConfigBackend?reset=1',   '{ts escape="sql" skip="true"}Cleanup Caches and Update Paths{/ts}', 'Cleanup Caches and Update Paths', 'administer CiviCRM', '', @systemSettingslastID, '1', NULL, 4 ),
+    ( @domainID, 'civicrm/admin/setting/updateConfigBackend?reset=1',   '{ts escape="sql" skip="true"}Clear Caches and Reset Paths{/ts}', 'Clear Caches and Reset Paths', 'administer CiviCRM', '', @systemSettingslastID, '1', NULL, 4 ),
```

Comment:
The `name` fields shouldn't change. (I wish `name` was called `internalLookupKey` or something like that and that the values were either integers or more computery, to be clear they are row ids and not actually meaningful english words, and that just happens to be from history when there was no label field.)